### PR TITLE
BTS-666 | Java driver: Add remarks about VelocyStream removal in 3.12

### PR DESCRIPTION
**Pull request by Simran-B**
#### Upstream PRs

- 3.10:
- 3.11:
- 3.12:

### CODE REVIEW

File: site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md
Comment (Simran-B) [inline site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md:1]: @rashtao I switched it around to show the most meaningful combination, which I assume is VPack over HTTP

Comment (rashtao) [reply site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md:1]: :+1:

File: site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md [hunk 1]
```diff
@@ -122,7 +123,7 @@ For example in Maven:
         <exclusions>
             <exclusion>
                 <groupId>com.arangodb</groupId>
-                <artifactId>http-protocol</artifactId>
+                <artifactId>vst-protocol</artifactId>
```
Comment (rashtao) [inline site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md:126]: This is about excluding the HTTP protocol module, so it should be:
```suggestion
                <artifactId>http-protocol</artifactId>
```

File: site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md [hunk 2]
```diff
@@ -132,7 +133,7 @@ For example in Maven:
     </dependency>
     <dependency>
         <groupId>com.arangodb</groupId>
-        <artifactId>vst-protocol</artifactId>
+        <artifactId>http-protocol</artifactId>
```
Comment (rashtao) [inline site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md:136]: This is about importing the VST protocol module, so it should be:
```suggestion
        <artifactId>vst-protocol</artifactId>
```

**Review by rashtao — CHANGES_REQUESTED** [review]

**Review by naushniki — APPROVED** [review]

File: site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md [hunk 3]
```diff
@@ -132,7 +133,7 @@ For example in Maven:
...
     </dependency>
```
Comment (rashtao) [inline site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md:137]: `http-protocol` is transitively imported, no need to re-add it here:
```suggestion
```

File: site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md [hunk 4]
```diff
@@ -96,20 +96,21 @@ The main driver artifact `com.arangodb:arangodb-java-driver` has transitive depe
...
 In case a non-default communication protocol or user serde are used, the related module(s)
 must be explicitly included and the corresponding default module(s) can be excluded.
 
-For example, to use the driver with `VPACK` over `VST`, we must include:
-- `com.arangodb:vst-protocol` and
-- `com.arangodb:jackson-serde-vpack`
+For example, to use the driver with `VPACK` over `HTTP`, you must include:
 
-and can exclude:
 - `com.arangodb:http-protocol` and
```
Comment (rashtao) [inline site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md:109]: `http-protocol` is transitively imported, no need to re-add it here:
```suggestion
```

File: site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md [hunk 5]
```diff
@@ -96,20 +96,21 @@ The main driver artifact `com.arangodb:arangodb-java-driver` has transitive depe
...
+- `com.arangodb:jackson-serde-vpack`
+
+and you can exclude:
+- `com.arangodb:vst-protocol` and
```
Comment (rashtao) [inline site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md:113]: `vst-protocol` is not imported, so no exclusion is needed:
```suggestion
```

File: site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md [hunk 6]
```diff
@@ -122,7 +123,7 @@ For example in Maven:
         <exclusions>
             <exclusion>
                 <groupId>com.arangodb</groupId>
-                <artifactId>http-protocol</artifactId>
+                <artifactId>vst-protocol</artifactId>
             </exclusion>
```
Comment (rashtao) [inline site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md:127]: `vst-protocol` is not imported, so no exclusion is needed:
```suggestion
```

**Review by rashtao — CHANGES_REQUESTED** [review]

**Comment by Simran-B** [comment]
@rashtao Is this good to go now?
https://github.com/arangodb/docs-hugo/blob/2c5dd516f71a7a74148fadf778c1810cf5fe02f6/site/content/3.12/develop/drivers/java/reference-version-7/changes-in-version-7.md#modules

**Review by rashtao — APPROVED** [review]

**Pull request merged**